example-get-started-experiments/code/.github/workflows/deploy-model-template.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Model (Template)
+
+on:
+  # the workflow is triggered whenever a tag is pushed to the repository
+  push:
+    tags:
+      - "*"
+  # we can also run the workflow manually...
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Manual version name'
+        required: true
+        type: string
+  
+permissions:
+  contents: write
+  id-token: write
```

Comment:
I think we could drop these AFAIK. Looking [here](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), write permissions to `contents` are granted by default (not sure if we need it anyway here), and `id-token` is only needed when using AWS roles to authorize via OIDC.

Reply:
Agreed. Again, this was a mindless copypasta...These should probably stay in the Sagemaker workflow, or at least the id-token. I'll try and see what happens if I remove one or both.

Reply:
Yeah, I'm fine to drop them here and leave in Sagemaker if you don't want to keep iterating on every combination here.

Reply:
Deleting them from template, keeping in sagemaker